webapp/packages/core-ui/src/Form/FormState.ts

```diff
@@ -94,6 +97,7 @@ export class FormState<TState> implements IFormState<TState> {
       setPartsState: action,
       setState: action,
       updateFormState: action,
+      save: action,
```

Comment:
This does nothing because `save` is asynchronous. Action works only for synchronous mutations. You can use `runInAction` instead (but probably not in this case)